web/client/components/QueryForm/GeometryDetails.jsx

```diff
@@ -51,10 +57,15 @@ const GeometryDetails = React.createClass({
 
         this.props.onChangeDrawingStatus("replace", undefined, "queryform", [feature]);
     },
-    onUpdateRadius(value, name) {
+    onUpdateCircle(value, name) {
         this.tempCircle[name] = parseFloat(value);
 
-        let circle = new ol.geom.Circle(this.tempCircle.center, this.tempCircle[name]);
+        let mapProj = this.props.map.projection;
+        let center = new ol.geom.Point([this.tempCircle.x, this.tempCircle.y]);
+        center = !this.props.useMapProjection ? center.clone().transform('EPSG:4326', mapProj) : center;
+
+        let coordinates = center.getCoordinates();
+        let circle = new ol.geom.Circle(coordinates, this.tempCircle.radius);
         circle = ol.geom.Polygon.fromCircle(circle, 100);
```

Comment:
Same as above. In alternative, GeometryDetails can become a mapping library dependent component